_opensearch/segment-replication/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Segment Replication Feature
+nav_order: 122
+parent: Opensearch
+has_children: true
+redirect_from:
+  - /opensearch/segment-replication/
+---
+
+## Segment Replication
+
+Opensearch 2.3 contains an experimental version of segment replication.
```

Comment:
do they? we're posting them in a new feature section, which is my understanding, but is otherwise just going to be on the site at large as we dropped having an "experimental feature" dropdown. My concern would be that a Google search would bring them to this page and they wouldn't necessarily know it was an experimental feature. Feedback appreciated on this.

Reply:
notes will be included on experimental feature page